core/Cargo.toml

```diff
@@ -12,6 +12,7 @@ repository = "https://github.com/wafflespeanut/paperclip"
 [dependencies]
 actix-web = { version = "2.0", optional = true }
 actix-http = { version = "1.0", optional = true }
+actix-multipart = { version = "0.2", optional = true }
```

Comment:
Does this work at the moment? There's no feature in `core` that ties this dependency.

Reply:
It seems that if there is no feature then plain optional dependency is activated. I haven't found any info in cargo documentation about this, but I've checked it and it works for me.

Reply:
Interesting. I guess this is why they don't allow feature name to collide with dependency name. :smile: